src/Mantasflowers.WebApi/appsettings.Development.json

```diff
@@ -6,7 +6,7 @@
     "Name": "mantas-flowers",
     "DataSource": "(localdb)\\mssqllocaldb",
     "WindowsAuthentication": "True",
-    "IsInMemory": "True"
+    "IsInMemory": "False"
```

Comment:
> 
> 
> Also regarding your comment about transactions and IN_MEMORY DB. It's not supposed to work with transactions and the warnings are purposively silenced: [DatabaseSetup.cs](https://github.com/mantasflowers/Mantasflowers.Backend/blob/master/src/Mantasflowers.WebApi/Setup/Database/DatabaseSetup.cs#L26)

That's exactly what I'm saying: in future it should be deprecated from DatabaseSetup.cs if transactions will remain.
```cs
if (sqlConfig.DatabaseConfiguration.IsInMemory)
{
    services.AddDbContext<DatabaseContext>(options =>
    {
        options.UseInMemoryDatabase("IN_MEMORY")
            .ConfigureWarnings(x =>
                x.Ignore(InMemoryEventId.TransactionIgnoredWarning))
            .ConfigureWarnings(x => 
                x.Throw(RelationalEventId.QueryPossibleUnintendedUseOfEqualsWarning));
    },
    ServiceLifetime.Transient,
    ServiceLifetime.Singleton);
}
```

Reply:
I wouldn't remove it. It's main purpose is for running in local env (which is very comfortable if you don't have DB configured or want fresh DB state). The behavior might obviously be different duo to transactions not working, but in normal case it will be the same.